crates/ide_completion/src/item.rs

```diff
@@ -107,9 +107,9 @@ impl fmt::Debug for CompletionItem {
         if self.deprecated {
             s.field("deprecated", &true);
         }
-        if self.relevance.is_relevant() {
-            s.field("relevance", &self.relevance);
-        }
+
+        s.field("relevance_score", &self.relevance.score());
```

Comment:
I think for debugging purposes it's more useful to see components of relevance, rather then the final score. For score, we need to just sort the variants

Reply:
ah, no, we have separate infra for relevance testing, using score is good then!

Reply:
I think you're right. When debugging a `CompletionItem` we'll likely want to know about the components of relevance. When debugging the scoring algorithm (within relevance) we'll care about score/ordering, which is what we check in that other infra. 

I went ahead and changed this.